applications/NXapm.nxdl.xml

```diff
@@ -319,21 +317,37 @@
                 included when the atom probe session started. If the exact duration of
                 the measurement is not relevant, start_time only should be used.
                 
-                Often though it is useful to specify both start_time and end_time to
-                capture more detailed bookkeeping of the experiment. The user should
-                be aware that even with having both dates specified, it may not be
-                possible to infer how long the experiment took or for how long data
-                were collected.
+                The start_time is required to have provide always at least one
+                point in time and thus temporal context to a measurement and simulation
+                when writing instance data using NXapm. Otherwise, the instance data
+                can not be sorted in order or even placed in a logical sequence to other
+                steps of the research workflow, which would disallow using functionalities
+                in research data management systems that rely on temporal context.
+                
+                Specifying start_time and end_time is useful for capturing more detailed
+                bookkeeping of the experiment. The user should be aware that even with
+                having both dates specified, it may not be possible to infer how long
+                the experiment took or for how long data were collected.
                 
                 More detailed timing data over the course of the experiment have to be
                 collected to compute this event chain during the experiment. For this
                 purpose the :ref:`NXevent_data_apm` instance should be used.
+                
+                Writing the end_time can be a tricky in practice. If written at the start
```

Comment:
This paragraph looks to be in the wrong place. I would expect it in the "end_time" field.